tools/create_db.py

```diff
@@ -34,9 +35,12 @@
 # must call digits.config.load_config() before caffe to set the path
 import caffe.io
 import caffe_pb2
+import mxnet as mx
```

Comment:
If we do add support for mxnet, it will certainly be optional. Not required for everyone like this.